server/spec/middlewares/websocket_backend_spec.rb

```diff
@@ -4,10 +4,23 @@
   let(:app) { spy(:app) }
   let(:subject) { described_class.new(app) }
 
+  before(:all) do
+    EM.run
```

Comment:
I had it like that and travis did not like it.. that's why it's now before/after :all .